### ISSUE TITLE
[Bug] Aeternium pickaxe with fortune III doesn't work on some blocks that can be influenced by fortune

### ISSUE THREAD

**Issue by aoligei635** [quiqueck/BetterEnd#248]
### What happened?

I was using a aeternium pickaxe enchanted fortune III.Later I found that I use the APWF to dig crystal clusters,they drop only 2 shards stably.But when I used a diamond pickaxe enchanted fortune III,a cluster dropped 12 SHARDS!

### BetterEnd

4.0.6

### BCLib

3.0.7

### Fabric API

0.83.0

### Fabric Loader

0.14.21

### Minecraft

1.20.0

**Comment by quiqueck** [comment]
Thanks for reporting. Was that one of the sulfur crystals in the end?

**Comment by aoligei635** [comment]
No.Sulfur crystal can be influenced normally by APWF,it can drop the same number as picked with DPWF.But amethyst crystals can not be influenced by APWF
